controlplane/src/core/repositories/FederatedGraphRepository.ts

```diff
@@ -548,6 +548,50 @@ export class FederatedGraphRepository {
     });
   }
 
+  public async fetchLatestFederatedGraphChangelog(
+    federatedGraphId: string,
+  ): Promise<FederatedGraphChangelogDTO | undefined> {
+    const federatedGraph = await this.db
+      .select({ schemaVersionId: federatedGraphs.composedSchemaVersionId })
+      .from(federatedGraphs)
+      .where(eq(federatedGraphs.id, federatedGraphId));
```

Comment:
Please always filter on organizationID to ensure the user has access to it.

Reply:
Done